src/test/timeout.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { equal } from "assert"
+import { parseTimeoutSeconds, defaultTimeout } from "../parseTimeoutSeconds"
+
+describe("parseTimeoutSeconds", () => {
```

Comment:
in case we have to keep our implementation, here is the reference in shocon:
https://github.com/akka-js/shocon/blob/master/facade/shared/src/main/scala/com/typesafe/config/Config.scala#L197

Reply:
I remember we used only the outputs of `scala.concurrent.duration`'s`toString` with seconds, minutes and hours, not the entire HOCON standard. The same we do in many tools (pylint, prospector, codacy-sonar-csharp, etc.)
cc. @DReigada

Reply:
This was a mistake on our side, the timeout should be set as a universal format (e.g. number of seconds) and not scala duration. We should just ignore the timeouts in tools that do not use the scala seed. And eventually in the future we will add a new env var which passes the timeout correctly

Reply:
I would say to leave this implementation until the PR to introduce the new `TIMEOUT_SECONDS` env variable is merged in codacy internals.. Then I will remove the timeout parsing implementation and switch to the new one.

Reply:
ok with this